doc.go

```diff
@@ -75,6 +75,7 @@ The following options can be used to customize the behavior of lf:
     globsearch  boolean  (default off)
     hidden      boolean  (default off)
     ignorecase  boolean  (default on)
+    lscolors    boolean  (default on if LSCOLORS or LS_COLORS environment variable is set)
```

Comment:
Is it really necessary to set the default value based on the existence of these variables? Maybe `lscolors` can be `on` by default and it can only take effect when these variables are set. Also another alternative is to have a `color` option instead of this `lscolors` option to toggle the use of colors altogether which I think would make more sense.

Reply:
I intentionally kept the original behavior as a fallback. Though it's nice to have ability to configure colors I believe there are many people out there who don't bother and just want "some colors".